charts/azdo-proxy/templates/networkpolicy.yaml

```diff
@@ -2,17 +2,43 @@
 apiVersion: networking.k8s.io/v1
 kind: NetworkPolicy
 metadata:
-  name: {{ include "azdo-proxy.fullname" . }}-deny-ingress
+  name: {{ include "azdo-proxy.fullname" . }}
   labels:
     {{- include "azdo-proxy.labels" . | nindent 4 }}
 spec:
+  podSelector:
+    matchLabels:
+      {{- include "azdo-proxy.labels" . | nindent 6 }}
   policyTypes:
     - Ingress
     - Egress
   ingress:
-  - from:
-    - podSelector: {}
+    - from:
+        - podSelector:
+            matchLabels:
+              app: source-controller
+      ports:
+        - port: 8080
```

Comment:
@phillebaba will this limit it to the flux-system namespace?

Reply:
Yes